monitoring/quickstart.rb

```diff
@@ -0,0 +1,40 @@
+require "google/cloud/monitoring"
+
+def quickstart
+  # [START monitoring_quickstart]
+
+  # TODO: Replace Google Cloud Platform project ID
+  project_id = "YOUR_PROJECT_ID"
+
+  # Instantiates a client
+  metric_service_client = Google::Cloud::Monitoring::Metric.new
+  # Using keyfile.json
+  # metric_service_client = Google::Cloud::Monitoring::Metric.new(credentials: 'keyfile.json')
```

Comment:
Please just use this as the sample code, and remove the version that doesn't pass credentials explicitly.

Also, please use double quotes for strings by default.

Reply:
Removed keyhole comments